README.md

```diff
@@ -2,42 +2,45 @@
+* [Unicode](https://unicode.org) processing and related services to support text written in all human languages.
+* APIs and implementations which support the specific cultural & socio-linguistic preferences, such as localized methods for displaying dates & times.
+* The ability for [Node.js](https://github.com/nodejs/node) and its related modules & applications to be translated into distinct human languages.
 
-#### What l10n means to us
-* The ability for [Node.js](https://github.com/nodejs/node) and its related modules & applications to be translated into distinct human languages. 
+#### What l10n (Localization) means to us
+* Making the Node.js project's API documentation, site, and tutorial content available in every language.
 * The responsibility to provide translators with a useful platform to work in.
 * The responsibility to provision Node.js with all l10n assets, through coordination with [Node.js](https://github.com/nodejs/node) core contributors.
 * Members of this group are not responsible for performing the act of translation itself.
 
-
 ## Summary of our Responsibilities
 1. Supporting l10n groups who are actively translating the content of [Node.js](https://github.com/nodejs/node) into their own languages.
 2. i18n support for [Node.js](https://github.com/nodejs/node), and its related initiatives.
-3. Ensuring i18n compliance with all common character encoding standards such as Unicode, and ECMA.
+3. Ensuring i18n compliance with all relevant standards such as [Unicode](https://unicode.org) and [ECMA-402](https://github.com/tc39/ecma402).
 4. Continual refinement and maintenance of the i18n Working Group's processes, platform service accounts, and related module repositories & source code.
 
 ### Our Current Trajectory
 Please read [this article](https://medium.com/the-node-js-collection/internationalizing-node-js-fe7761798b0a) to get up to speed on our current trajectory.
 
 
 ## Inter-project relationships
-In order to ensure best practices, this working group may work directly with representatives of similar i18n efforts from other external projects. For example:
+In order to ensure best practices, this working group may work directly with representatives of similar and related i18n efforts from other external projects. For example:
 * [Electron i18n](https://github.com/electron/electron-i18n)
 * [W3C i18n Working Group](https://www.w3.org/International/core/Overview)
-* [V8 i18n](https://github.com/v8/v8/wiki/i18n-support)
+* [ECMA-402](https://github.com/tc39/ecma402) (Together with ECMA-262, specifies the i18n features of JavaScript)
+* [V8 i18n](https://github.com/v8/v8/wiki/i18n-support) (implements ECMA-402 in Node.js)
+* [Unicode](https://unicode.org):
+  - [The Unicode Standard](https://unicode.org/standard/) (defines how text is encoded)
+  - [CLDR](https://cldr.unicode.org) (Common Locale Data Repository)
+  - [ICU](https://icu-project.org) (C/C++ i18n library used by v8 and Node.js) 
```

Comment:
Thanks for adding CLDR/ICU to our relationships. Excited to support these efforts. 👍